tests/console/verify_config_files.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Module allows validation of generic configuration files.
+# Test data should be defined in the following format:
+#  configuration_files:
+#    - path: /etc/hosts
+#      entries:
+#        - 'new.entry.de\t10.226.154.19 h999uz'
+#    - path: /etc/chrony.conf
+#      entries:
+#        - pool ntp.suse.de iburst
+# NOTE: grep -P is used for validation, therefore can be used in entries
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_module_consoletest';
+use strict;
+use warnings;
+use testapi;
+use scheduler;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    my $test_data = get_test_suite_data();
+    # Accumulate errors for all checks
+    my $errors = '';
+    foreach my $file (@{$test_data->{configuration_files}}) {
+        foreach my $entry (@{$file->{entries}}) {
+            if (script_run("grep -P \"$entry\" $file->{path}") != 0) {
```

Comment:
did you find any advantage to do it line by line using `grep -P` instead of using `grep` in Perl doing it in one shot and processing the output? I'd like to consider it for following-up ticket that you mentioned, which we could create in this epic https://progress.opensuse.org/issues/71242 to unify with [check config files in kdump](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/yast2_kdump.pm#L40)

Reply:
The main problem there is gathering the output from the system over serial, as it still has some chances to get polluted, etc. So this approach is consistent from this perspective, but slower as we need to do more typing.
I will file the ticket later today.